README.md

```diff
@@ -163,7 +163,7 @@ The below code demonstrates how to re-instantiate a Wallet from the data export.
 ```ruby
 # The Wallet can be re-instantiated using the exported data.
 # w4 will be equivalent to w3.
-w4 = u.import_wallet(data)
+w4 = Coinbase::Wallet.import(data)
```

Comment:
update line 174 to `w5 = wallets[w3.id]`

Reply:
Can we make the README / example something that is actually evaluated?